design-system/react/src/components/InputAmount/InputAmount.tsx

```diff
@@ -73,57 +82,97 @@ export const InputAmount: InputAmountComponent = ({
     }
   };
 
+  const tokenImage = (name?: string, imageUrl?: string) => {
+    return (
+      <Image
+        alt={name}
+        src={imageUrl}
+        width={14}
+        height={14}
+        css={styles.image}
+      />
+    );
+  };
+
   return (
     <Input size="lg" css={styles.input} {...props}>
-      <Input.Number
-        autoComplete="off"
-        inputMode="decimal"
-        name={name}
-        aria-label={name}
-        placeholder="0.00"
-        allowedDecimalSeparators={['.', ',']}
-        allowNegative={false}
-        thousandSeparator={false}
-        value={assetAmount}
-        onChange={(e) => {
-          handleAmountChange(e.target.value);
-        }}
-        decimalScale={units}
-        {...inputProps}
-      />
-      {initialBalance && (
-        <Input.ElementRight css={styles.elementRight}>
-          <Box css={styles.balanceActions}>
-            {!hiddenMaxButton && (
-              <Flex align="center" justify="end">
+      <Text color="intentsBase9" css={styles.heading}>
+        Amount
+      </Text>
+      <Flex align="center" css={styles.secondRow}>
```

Comment:
Try to avoid mixing things like here, instead of defining `align` here, put this definition inside `styles.secondRow`